src/components/SearchBar/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex, Input, InputGroup, InputLeftElement } from '@chakra-ui/react';
+import { MdOutlineSearch } from 'react-icons/md';
+
+export const SearchBar = () => {
+  return (
+    <Flex m="1.6rem 4rem">
+      <InputGroup size="lg">
+        <InputLeftElement pointerEvents="none">
+          <MdOutlineSearch color="#34333A" fontSize={22} />
+        </InputLeftElement>
+
+        <Input
+          border={'none'}
+          placeholder="search item"
+          _placeholder={{
+            color: 'rgba(189, 189, 189, 1)',
+            fontSize: 15,
```

Comment:
Change de values to be equal to figma

```suggestion
            color: '#BDBDBD',
            fontSize: 14,
```